packages/auth-express/src/index.ts

```diff
@@ -0,0 +1,568 @@
+          await this.core
+        ).signinWithEmailPassword(email, password);
+        res.cookie(this.options.authCookieName, tokenData.auth_token, {
+          httpOnly: true,
+          sameSite: "strict",
+        });
+        req.session = new ExpressAuthSession(this.client, tokenData.auth_token);
+        req.tokenData = tokenData;
+        next();
+      } catch (err) {
+        next(err);
+      }
+    },
+    signUp:
+      (verifyUrl: string) =>
+      async (req: AuthRequest, res: ExpressResponse, next: NextFunction) => {
+        try {
+          const [email, password] = _extractParams(
+            req.body,
+            ["email", "password"],
+            "email or password missing from request body"
+          );
+          const result = await (
+            await this.core
+          ).signupWithEmailPassword(email, password, verifyUrl);
+          res.cookie(this.options.pkceVerifierCookieName, result.verifier, {
+            httpOnly: true,
+            sameSite: "strict",
+          });
+          if (result.status === "complete") {
+            res.cookie(
+              this.options.authCookieName,
+              result.tokenData.auth_token,
+              {
+                httpOnly: true,
+                sameSite: "strict",
+              }
+            );
+            req.session = new ExpressAuthSession(
+              this.client,
+              result.tokenData.auth_token
+            );
+            req.tokenData = result.tokenData;
+          } else {
+            req.session = new ExpressAuthSession(this.client, undefined);
+          }
+          next();
+        } catch (err) {
+          next(err);
+        }
+      },
+    verify: async (
+      req: AuthRequest,
+      res: ExpressResponse,
+      next: NextFunction
+    ) => {
+      try {
+        const searchParams = new URLSearchParams(req.url.split("?")[1]);
+        const verificationToken = searchParams.get("verification_token");
+        const verifier = req.cookies[this.options.pkceVerifierCookieName];
+        if (!verificationToken) {
+          throw new Error("no verification_token in response");
+        }
+        if (!verifier) {
+          throw new Error("no pkce verifier cookie found");
+        }
+        const tokenData = await (
+          await this.core
+        ).verifyEmailPasswordSignup(verificationToken, verifier);
+        res.cookie(this.options.authCookieName, tokenData.auth_token, {
+          httpOnly: true,
+          sameSite: "strict",
+        });
+        res.clearCookie(this.options.pkceVerifierCookieName);
+
+        req.session = new ExpressAuthSession(this.client, tokenData.auth_token);
+        req.tokenData = tokenData;
+        next();
+      } catch (err) {
+        next(err);
+      }
+    },
+    sendPasswordResetEmail:
+      (passwordResetUrl: string) =>
+      async (req: AuthRequest, res: ExpressResponse, next: NextFunction) => {
+        try {
+          if (!this.options.passwordResetPath) {
```

Comment:
We don't seem to use this option, so maybe we don't need it for the express integration, and the user can just pass it directly when they're setting up the middleware?